bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +41,36 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        line_nr_start = next((i for i, line in reversed(list(
+            enumerate(file, start=1))) if line.strip() != ''), 0)
```

Comment:
Optional:
Is it maybe possible to wrap the statement somehow so it's more readable?

Reply:
Something in the direction of
```python
line_nr_start = next(
    (i
     for i, line in reversed(list(enumerate(file, start=1)))
     if line.strip() != ''), 0)
```